pkgs/development/tools/misc/libtool/libtool2.nix

```diff
@@ -8,20 +10,16 @@
 
 stdenv.mkDerivation rec {
   pname = "libtool";
-  version = "2.4.6";
+  version = "2.4.7";
 
   src = fetchurl {
     url = "mirror://gnu/libtool/${pname}-${version}.tar.gz";
-    sha256 = "1qq61k6lp1fp75xs398yzi6wvbx232l7xbyn3p13cnh27mflvgg3";
+    sha256 = "sha256-BOlsJATqcMWQxUbrpCAqThJyLGQAFsErmy8c49SB6ag=";
   };
 
   outputs = [ "out" "lib" ];
 
-  patches = [
-    # Suport macOS version 11.0
-    # https://lists.gnu.org/archive/html/libtool-patches/2020-06/msg00001.html
-    ./libtool2-macos11.patch
-  ];
+  FILECMD = "${file}/bin/file";
```

Comment:
Is that used by autoconf? Maybe add a short comment to it.

Reply:
Yes; the `FILECMD` was introduced in libtool 2.4.7 in order to fix the fact that `/usr/bin/file` was a hardwired path in earlier versions.  I will add a comment.